client/landing/stepper/declarative-flow/site-setup-flow.ts

```diff
@@ -479,6 +479,13 @@ const siteSetupFlow: Flow = {
 					return navigate( 'vertical' );
 
 				case 'patternAssembler':
+					// eslint-disable-next-line no-case-declarations
+					const backTo = urlQueryParams.get( 'backTo' );
+
+					if ( backTo ) {
+						return redirect( backTo );
+					}
+
```

Comment:
@Automattic/lego same here, this might also conflict with your work done in https://github.com/Automattic/wp-calypso/pull/71852. This was needed to allow users to go back to the themes showcase from the pattern assembler flow if they click the back button. Do you see any issues with this approach?

Reply:
I don't see any issues. Thanks for sharing it! We could use it too.